doc/source/tune/_tutorials/tune-distributed.rst

```diff
@@ -130,7 +130,7 @@ If you used a cluster configuration (starting a cluster with ``ray up`` or ``ray
 Syncing
 -------
 
-Tune automatically syncs the trial folder on remote nodes back to the head node. This requires the ray cluster to be started with the :ref:`cluster launcher <ref-automatic-cluster>`.
+Tune automatically syncs the trial folder on remote nodes back to the head node. This requires the ray cluster to be started with the :ref:`cluster launcher <cluster-cloud>`.
```

Comment:
we keep saying "the cluster launcher". does this just mean "ray up"? we should just say that if it is...

Reply:
the cluster launcher is everything that happens before the head node and until the head node is running. The autoscaler is everything that happens after to launch workers.